conda-envs/windows-environment-dev-py38.yml

```diff
@@ -0,0 +1,26 @@
+name: pymc3-dev-py38
```

Comment:
Same comment as the other PR. Just add a docstring explaining the nuances of "what does windows do different"

Reply:
Deferred to #4786